lldb/include/lldb/Target/Process.h

```diff
@@ -1469,6 +1468,14 @@ class Process : public std::enable_shared_from_this<Process>,
 
   virtual bool IsLiveDebugSession() const { return true; };
 
+
+  /// Provide a way to retrieve the core dump file that is loaded for debugging.
+  /// Only available if IsLiveDebugSession() returns true.
```

Comment:
Shouldn't it only be available if `IsLiveDebugSession` is false? You're debugging a core dump which isn't really a "live debug session" since you're examining the corpse of a process, no?